src/collective/lazysizes/transform.py

```diff
@@ -157,19 +157,22 @@ def transformUnicode(self, result, encoding):
 
     def transformIterable(self, result, encoding):
         registry = getUtility(IRegistry)
-        settings = registry.forInterface(ILazySizesSettings)
+        settings = registry.forInterface(ILazySizesSettings, check=False)
 
         # we apply the transform always for anonymous users
         if not api.user.is_anonymous():
             # the user is authenticated, check if enabled
-            if not settings.lazyload_authenticated:
+            if not getattr(settings, 'lazyload_authenticated', False):
                 return  # transform not enabled for authenticated users
 
         result = self._parse(result)
         if result is None:
             return
 
-        blacklist = self._blacklist(result, settings.css_class_blacklist)
+        blacklist = self._blacklist(
+            result,
+            getattr(settings, 'css_class_blacklist', set())
```

Comment:
Also not strictly necessary, but I use the same pattern to access the settings like above...